scribble/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,46 @@
+class CommentsController < ApplicationController
+  def show
+    @post = Post.find(params[:post_id])
+    @comment = Comment.find(params[:id])
+  end
+
+  def edit
+    @post = Post.find(params[:post_id])
+    @comment = Comment.find(params[:id])
+  end
+
+  def new
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new
+  end
+
+  def update
+    @post = Post.find(params[:post_id])
+    @comment = Comment.find(params[:id])
+    @comment.update(comments_params)
+
+    redirect_to post_comment_path(@post, @comment)
```

Comment:
If every comment has a unique ID, then I would make a route for comments so that you can just `redirect_to @comment`. Ignoring all the "favorites" stuff, this provides an example of what I'm talking about:

https://github.com/ga-dc/tunr_rails_deployment/blob/master/config/routes.rb